WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2248,6 +2251,27 @@ static GSourceFuncs _handlerIntervention =
                 reinterpret_cast<GCallback>(wpeNotifyWPEFrameworkMessageReceivedCallback), this);
             webkit_user_content_manager_register_script_message_handler(userContentManager, "wpeNotifyWPEFramework");
 
+            if (_config.UserScripts.IsSet()) {
+                auto loadScript =
+                    [&userContentManager](const std::string& path) {
+                        gchar* scriptContent;
+                        auto success = g_file_get_contents(path.c_str(), &scriptContent, nullptr, nullptr);
+                        if (!success) {
+                            SYSLOG(Trace::Error, (_T("Unable to read user script '%s'"), path.c_str()));
+                            return;
+                        }
+                        auto* script = webkit_user_script_new(scriptContent, WEBKIT_USER_CONTENT_INJECT_ALL_FRAMES, WEBKIT_USER_SCRIPT_INJECT_AT_DOCUMENT_START, nullptr, nullptr);
+                        webkit_user_content_manager_add_script(userContentManager, script);
+                        webkit_user_script_unref(script);
+                        g_free(scriptContent);
+                    };
+                for (unsigned userScriptIndex = 0; userScriptIndex < _config.UserScripts.Length(); userScriptIndex++) {
+                    const auto& scriptPath = _config.UserScripts[userScriptIndex];
+                    SYSLOG(Trace::Information, (_T("Loading user script '%s'"), scriptPath.Value().c_str()));
```

Comment:
Hi, for the error message above it is consistent with other error message in this file to use SYSLOG however for this line  where it is informational I would propose to use "TRACE(Trace::Information, ..." here so we do not pollute the syslog output with informational messages